.travis.yml

```diff
@@ -4,7 +4,7 @@ python:
   - 2.7
   - 3.6
 addons:
-  firefox: 53.0
+  firefox: 54.0
```

Comment:
Thanks, indeed.  I think (and @davehunt can confirm, though), that we probably want to pin.  I'm honestly not sure, so kept it the safe way, for now :-)